src/format/Format.cc

```diff
@@ -508,7 +508,7 @@ Format::Format::assemble(MemBuf &mb, const AccessLogEntry::Pointer &al, int logS
 
         case LFT_LOCAL_LISTENING_PORT:
             if (const auto port = FindListeningPortNumber(nullptr, al.getRaw())) {
-                outint = port;
+                outint = *port;
```

Comment:
I am skipping this secondary change request until its [parent](https://github.com/squid-cache/squid/pull/1255/files#r1095565843) change request is resolved. FWIW, you can resolve this change request now because the _compiler_ will force me to make this adjustment after/if the code is adjusted the way that parent change request suggests.